agents/components/Profile.js

```diff
@@ -105,11 +105,6 @@ export default compose(
     toggleEdit: ({ setEditing }) => () => setEditing(not)
   }),
   connectForm({
-    form: 'profile',
-    initialValues: {
```

Comment:
right, ideally we pass it in from [here](https://github.com/root-systems/cobuy/blob/98cb9c859cbf201577f5dd0d3cdc97a842073068/agents/stories/Profile.js). i think we did something about this in #120, one sec.